docs/json.md

```diff
@@ -410,6 +410,62 @@ control over the resulting JSON object:
 
 <!--- TEST -->
 
+It is also possible to specify different class discriminators for different hierarchies. Instead of Json instance property, use [JsonClassDiscriminator] annotation directly on base serializable class:
```

Comment:
```suggestion
It is also possible to specify different class discriminators for different hierarchies. Instead of Json instance property, it is possible to use [JsonClassDiscriminator] annotation directly on base serializable class:
```

Reply:
'it is possible to' phrase repeating twice, it may be inconvenient to read